arangod/Agency/AsyncAgencyComm.cpp

```diff
@@ -432,6 +432,20 @@ arangodb::AsyncAgencyComm::FutureResult agencyAsyncSend(
 
 namespace arangodb {
 
+futures::Future<consensus::index_t> AsyncAgencyComm::getCurrentCommitIndex()
+    const {
+  auto future = sendWithFailover(fuerte::RestVerb::Get, "_api/agency/config",
+                                 120s, RequestType::CUSTOM, {});
```

Comment:
Question: if we use `RequestType::CUSTOM` here, we don't get a retry in case of errors or even redirects. Is that intentional? If not, we may want to use `RequestType::READ`.